packages/alice/pubspec.yaml

```diff
@@ -3,6 +3,7 @@ description: Alice is an HTTP Inspector tool which helps debugging http requests
 version: 1.0.0-dev.9
 homepage: https://github.com/jhomlala/alice
 repository: https://github.com/jhomlala/alice
+topics: http, inspector, dio, chopper, interceptor, alice
```

Comment:
You can have a max of 5 and they need to be a list, i.e.
```yaml
topics:
  - http
  - inspector
  - dio
  - chopper
  - interceptor
```

Reply:
Yup, forgot about syntax.